cmd/jindo/app/jindo.go

```diff
@@ -82,6 +85,8 @@ func init() {
 	jindoCmd.Flags().IntVar(&maxConcurrentReconciles, "runtime-workers", 3, "Set max concurrent workers for JindoRuntime controller")
 	jindoCmd.Flags().BoolVar(&eventDriven, "event-driven", true, "The reconciler's loop strategy. if it's false, it indicates period driven.")
 	jindoCmd.Flags().StringVarP(&pprofAddr, "pprof-addr", "", "", "The address for pprof to use while exporting profiling results")
+	jindoCmd.Flags().IntVarP(&restConfigQPS, "rest-config-qps", "", 20, "")     // 20 is the default qps in controller-runtime
```

Comment:
I suggest using `kube-api-qps` and `kube-api-burst` instead.

Reply:
Done. Following the naming convention in [KCM](
https://github.com/kubernetes/kubernetes/blob/3327bb367490d5ffa01eaa012253c478acbfdf52/staging/src/k8s.io/controller-manager/options/generic.go#L62-L63)